dashboards/victoriametrics.json

```diff
@@ -5256,7 +5256,7 @@
         },
         "definition": "label_values(vm_app_version{job=~\"$job\"}, instance)",
         "hide": 0,
-        "includeAll": false,
+        "includeAll": true,
```

Comment:
The rest of the panels may not have proper aggregation in queries for this. For example, panel ActiveTimeSeries will show two lines with identical names after this change
<img width="924" alt="image" src="https://github.com/VictoriaMetrics/VictoriaMetrics/assets/2902918/7b7d64af-5546-43c6-b73f-7f6f7a19a8dc">